js/lib/higlass-jupyter.js

```diff
@@ -21,6 +21,12 @@ var HiGlassDisplayModel = widgets.DOMWidgetModel.extend({
 // Custom View. Renders the widget model.
 var HiGlassDisplayView = widgets.DOMWidgetView.extend({
   render: function() {
+    var viewConfig = this.model.get('viewconf');
+    var height = this.model.get('height');
+    var hgOptions = this.model.get('hg_options');
+
+    var borderColor = hgOptions.isDarkTheme ? '#333333' : '#dddddd'
```

Comment:
Those values could be CONSTANTS but up to you if you want to change them.

Reply:
I tend to only create variables if they are used more than once or for readability. I don't really a benefit of creating two extra variables here. If the code gets more complex in the future we can change that of course.